python-spec/src/somacore/data.py

```diff
@@ -195,8 +200,12 @@ def create(
         :param uri: The URI where the array will be created.
         :param type: The Arrow type to store in the array.
             If the type is unsupported, an error will be raised.
-        :param shape: The length of each dimension as a sequence,
-            e.g. ``(100, 10)``. All lengths must be in the postive int64 range.
+        :param shape: The maximum capacity of each dimension, including room
+            for any intended future appends, as a sequence.  E.g. ``(100, 10)``.
+            All lengths must be in the postive int64 range, or `None`.  If a
+            slot is `None` -- only supported for `SparseNDArray`, not
+            `DenseNDArray` -- then the maximum possible int64 will be used.
+            This makes a `SOMASparseNDArray` growable.
```

Comment:
I would write this in such a way that it does not interrupt the sentence, maybe:

"For ``SparseNDArray`` only, if a slot is ``None``, ..."

or

"If a slot is ``None``, ... Empty slots are supported only by ``SparseNDArray``."